neuralmagic/sparseml
# [Feature branch] openpifpaf integration

Base feature  branch

Manual tests:
1. Ran the training commands listed in the README with yaml file shown in readme and verified sparsity levels changed via log output
2. Ran the export commands listed in the README with the checkpoint from training command & verified output is runnable with deepsparse.benchmark
3. Ran the training command with --checkpoint instead of --basenet arg from a non-finished training epoch to verify resume logic
4. Ran the training command with --checkpoint instead of --basenet from a finished training epoch to verify checkpoint logic
